.pre-commit-config.yaml

```
@@ -9,8 +9,8 @@ repos:
     hooks:
       - id: flake8
 
-  - repo: https://github.com/pre-commit/mirrors-isort
-    rev: v4.3.21
+  - repo: https://github.com/timothycrosley/isort
+    rev: 4.3.21
```

```suggestion
    rev: 4.3.21-2
```

Because they got the tag wrong the first time, IIUC.